### DISCUSSION TITLE
How to get address from specific location (not in locales)

### DISCUSSION THREAD

**Discussion by GustavoDcs** [faker-js/faker#1775] [Q&A]
Is there an easy way to get the address for a specific location that is not present inside the locales? 
For example: Colombia. I noticed that there is no es_CO, is there an alternative to it?

**Comment by ST-DDT** [comment]
The recommendet way is to create a PR that adds the missing data.
However you can also add the data only locally.

````ts
const faker = new Faker({ locales: { es_CO: { location: { street_name: [...] } } }, locale: "es_CO" });
````

Another alternative is using just a similar locale such as `es`.

Does that answer your questions?
